### PR TITLE
Final Project Entropic maps BARRIERE PELTIER 

### PR DESCRIPTION
**Pull request by HugoPELTIER**

### CODE REVIEW

**Comment by marcocuturi** [comment]
Thanks! 

can you fix the linter issues? 

you can check first the [CONTRIBUTING.md](https://github.com/ott-jax/ott/blob/795fabeaaf294c774cda204d153e19958d0db625/.github/CONTRIBUTING.md?plain=1#L4) file and install `pre-commit`, and run it on your files.

Thanks!

**Comment by HugoPELTIER** [comment]
Dear Mister Cuturi,

We apologise for the inconvenience. We have sorted out the problems with the commit and we applied pre-commit. Thank you in advance. Sincerely yours,

**Comment by HugoPELTIER** [comment]
I am very sorry, it seems that the remaining errors are due to spell checks. In particular, the names of individuals seem to be considered as spell errors.

File: docs/tutorials/notebooks/EntropicMaps_Barriere_Peltier.ipynb [hunk 1]
```diff
@@ -0,0 +1,1562 @@
+{
```
Comment (marcocuturi) [inline docs/tutorials/notebooks/EntropicMaps_Barriere_Peltier.ipynb:1]: <p>typos:</p><p>"based on the results of presented in"</p><p>"We considere"</p><p>"given the two probability distributions P and Q," -&gt; given two, P, Q should be written in markdown</p><p>REF</p><p> </p><p>idea: is the idea really to let \varepsilon go to 0?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (marcocuturi) [inline docs/tutorials/notebooks/EntropicMaps_Barriere_Peltier.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #15.    entropic_pot = potentials.EntropicPotentials(
</pre><p>this is easier by just instantiating <code>ot_sink.to_dual_potentials</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Pull request closed without merging**